e2e/verifiable-credentials.service.e2e.ts

```diff
@@ -220,4 +285,29 @@ describe('Verifiable credentials tests', () => {
       });
     });
   });
+
+  describe('Verifiable credentials exchange', () => {
+    const exchangeUrl = 'exchange url with exchange id';
+
+    test('initiateExchange() should return presentation matching presentation request', async () => {
+      const vc = await createExampleSignedCredential([]);
+      (axios as jest.Mocked<typeof axios>).post.mockImplementationOnce(
+        (url) => {
+          return Promise.resolve({
+            data: url === exchangeUrl ? vpRequest : '',
+          });
+        }
+      );
+      mockGetCredentialsByDefinition.mockImplementationOnce((vpDefinition) =>
```

Comment:
I think this is not good idea to mock also storage client methods. Mocking `axios` should be enough

Reply:
I would prefer to mock only direct dependencies of storage-client. Otherwise we will need to change tests if implementation of storage-client changed